popiska_v56.txt

```diff
@@ -0,0 +1 @@
+Toto je soubor ve vetvi v56
```

Comment:
to je divne, ze nevidim odstranenou versi

Reply:
to je dano tim, ze porovnavame s master a tam je prazdny soubor